plutus-conformance/test-cases/uplc/evaluation/term/constant-case/bool/bool-01/bool-01.uplc

```diff
@@ -0,0 +1,4 @@
+-- case on false
+(program 1.1.0
+    (case (con bool False) (con integer 0) (con integer 1))
+)
```

Comment:
The choice of constants is kinda unfortunate: what if `case` simply converts the given boolean to the respective integer instead of going into the right branch? It'd be better to make it more random, like return 69 and 42 or whatever.